tests/testthat/_snaps/package.md

```diff
@@ -20,3 +20,13 @@
       ! template.bootstrap must be 3 or 5, not 1.
       i Edit _pkgdown.yml to fix the problem.
 
+# read_meta() errors gracefully if _pkgdown.yml failed to parse
+
+    Code
+      read_meta(file, check_path = FALSE)
+    Condition
+      Error:
+      x Could not parse the config file.
+      !  Parser error: while parsing a block mapping at line 1, column 1 did not find expected key at line 9, column 3
```

Comment:
Looks like there is a space before "Parser" in the error that could be stripped

Reply:
Thanks!